DolphinVM.sln

```diff
@@ -94,21 +94,27 @@ Global
 		{CD85B8FA-6C9E-45C0-AFC8-0A79A7E5D0F7}.Release|Win32.ActiveCfg = Release|Win32
 		{CD85B8FA-6C9E-45C0-AFC8-0A79A7E5D0F7}.Release|Win32.Build.0 = Release|Win32
 		{CD85B8FA-6C9E-45C0-AFC8-0A79A7E5D0F7}.VM Debug|Win32.ActiveCfg = VM Debug|Win32
+		{CD85B8FA-6C9E-45C0-AFC8-0A79A7E5D0F7}.VM Debug|Win32.Build.0 = VM Debug|Win32

```

Comment:
I guess it's a matter of taste.
I did this because the debug build has been broken a couple of times now,
and I missed some errors caused in the stubs because they were excluded.
But now I know that, I can easily change the settings after a new clone.
.
I will do the PR onze more without the settings changes.
Should anything still be wrong, please implement the (trivial) fixes in own enviroment.
.
I totally agree that the CI should also always build the *full* debug build and report if it fails.
Is that possible with the stubs-disabled solution? Can I help with that?

Reply:
Don't worry. I'll merge your change and apply the CI change. It seems it may be quite straightforward in fact, with a little appveyor knowledge.

Reply:
Nope, I'll apply it, then adjust the config.